testing/test_kerns.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 import numpy as np
 import unittest
-from .reference import referenceRbfKernel, referencePeriodicKernel
+from testing.reference import referenceRbfKernel, referencePeriodicKernel
```

Comment:
I can run the single test using `python -m testing.test_kerns` as well though. So I'm happy either way. Any strong opinion?